build-system/server/autocomplete-test-data.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const mentionEmails = [
+  'harrypotter@hogwarts.edu',
+  'albusdumbledore@hogwarts.edu',
+  'voldemort@deatheater.org',
```

Comment:
I was initially under the assumption that there was such a thing as a "death theater", and the email address was contracting the duplicate "th". Then I went down a wikipedia rabbit hole.

Now, I'm wondering if you should include some folks from `muggleborn.biz` and `ministryofmagic.gov`.